### ISSUE TITLE
README unclear post example with syntax error

### ISSUE THREAD

**Issue by mykhal** [raku-community-modules/LWP-Simple#47]
It's not obvious what `so` param should mean in [README](https://github.com/raku-community-modules/LWP-Simple/blob/master/README.md#Synopsis) `Synopsis post` method example.

Anyway, it's missing final `);`.

**Comment by mykhal** [comment]
After syntax fix, the HTTP request produced is:

```
POST / HTTP/1.1
Connection: close
Host: 127.0.0.1
User-Agent: LWP::Simple/0.107 Raku/rakudo
so: True
```
so, the `so` param is used as additional HTTP header, which might be unexpected.

**Comment by Altai-man** [comment]
You are correct, certainly a better example should be shown, something obvious like `header1 => ..., header2 => ...` is best.

**Comment by mykhal** [comment]
I think yet better would be POST param(s) instead of custom header(s). ;-)

**Comment by Altai-man** [comment]
`content` is a third argument, see https://github.com/raku-community-modules/LWP-Simple/blob/master/lib/LWP/Simple.pm#L54-L56
The module certainly needs a maintainer anyway.
